modules/heat_conduction/test/tests/radiative_bcs/tests

```diff
@@ -7,4 +7,13 @@
     design = 'source/bcs/InfiniteCylinderRadiativeBC.md'
     issues = "#13053"
   [../]
+  [./function_radiative_bc]
+    type = Exodiff
+    input = 'function_radiative_bc.i'
+    exodiff = 'function_radiative_bc_out.e'
+    requirement = "MOOSE shall be able to model radiative heat transfer using a user-specified emissivity function."
+    design = 'source/bcs/FunctionRadiativeBC.md'
+    issues = "#13053"
```

Comment:
```suggestion
    issues = "#13053 #16962"
```

Reply:
I'm curious:  are we supposed to put PR ids in all new issues lines?  I thought they were for Issues only.

Reply:
Crud. This was what happens when I review late in the evening... you're absolutely correct.